src/Symfony/Component/Notifier/Bridge/GoogleChat/README.md

```diff
@@ -14,7 +14,7 @@ where:
  - `ACCESS_KEY` is your Google Chat access key
  - `ACCESS_TOKEN` is your Google Chat access token
  - `SPACE` is the Google Chat space
- - `THREAD_KEY` is the Google Chat message thread to group messages into a single thread (optional)
+ - ~~`THREAD_KEY` is the Google Chat message thread to group messages into a single thread (optional)~~ (deprecated)
```

Comment:
Why deprecating this feature? I think it's still a good option to be able to choose a default thread for all notifications.

Reply:
I assumed this DSN should closely match the actual API's endpoint. This has been reverted back.